mod/messageboard/views/default/elgg/messageboard.js

```diff
@@ -8,7 +8,7 @@ define(function (require) {
 	var MB = {
 		init: function() {
 			var form = $('form[name=elgg-messageboard]');
-			form.on('click', 'input[type=submit]', MB.submit);
+			form.on('click', 'button[type=submit]', MB.submit);
```

Comment:
main plan was to get this working again :) i did not want to refactor this js too much... as there should be an end to all the work :)